scripts/upload-image.sh

```diff
@@ -15,14 +13,13 @@ echo DESC: $DESC
 
 # enviroment variables
 SEABUG_ADMIN_TOKEN=ADMIN_TOKEN
```

Comment:
why rename?

Reply:
instead we should check that it is present

Reply:
what do you mean? what have I renamed?